lms/djangoapps/api_manager/courses/tasks.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from celery import task
+from django.conf import settings
+from django.core.cache import cache
+
+@task(name=u'lms.djangoapps.api_manager.courses.tasks.cache_static_tab_content')
+def cache_static_tab_contents(cache_key, contents):
+    """
+    Caches course static tab contents.
+    """
+    cache_expiration = getattr(settings, 'STATIC_TAB_CONTENTS_CACHE_MAX_AGE', 60 * 5)
```

Comment:
nit: please rename `STATIC_TAB_CONTENTS_CACHE_MAX_AGE` to `STATIC_TAB_CONTENTS_CACHE_TTL`